engine/src/widget.cpp

```diff
@@ -668,6 +668,28 @@ bool MCWidget::setcustomprop(MCExecContext& ctxt, MCNameRef p_set_name, MCNameRe
         !MCExtensionConvertFromScriptType(ctxt, t_set_type, InOut(t_value)))
```

Comment:
Although LCB will now throw an error, it will suffer the same fate as the type error problem - as the error is thrown before any LCB code is executed, so there is no LCB stack frame to generate an error on. So here if t_set_type == nullptr, then we should LegacyThrow 'EE_PROPERTY_CANTSET'.